.github/CODEOWNERS

```
@@ -0,0 +1,20 @@
+# anything with no explicit code owners will be tagged to @core-devs
+*                       @napari/core-devs
+
+# submodules
+napari/_vispy/          @brisvag @melonora
+napari/_qt/             @Czaki @lucyleeow @DragaDoncila
+napari/_app_model/      @lucyleeow
+napari/benchmarks/      @Czaki
+napari/plugins/         @Czaki @DragaDoncila @lucyleeow
+napari/qt/              @Czaki
```

Yeah there should never be a section with a single core dev 😬 (Thanks @DragaDoncila for stepping up in the places where @lucyleeow was listed on her own. 😅)

```suggestion
napari/benchmarks/      @Czaki @jni
napari/plugins/         @Czaki @DragaDoncila @lucyleeow
napari/qt/              @Czaki @jni
```